server/src/test/java/org/elasticsearch/common/util/CancellableSingleObjectCacheTests.java

```diff
@@ -65,22 +65,22 @@ public void testListenersCompletedByRefresh() {
         assertFalse(future1.isDone());
         testCache.assertPendingRefreshes(1);
         testCache.completeNextRefresh("foo", 1);
-        assertThat(future0.actionGet(0L), equalTo(1));
-        assertThat(future0.actionGet(0L), sameInstance(future1.actionGet(0L)));
+        assertThat(future0.result(), equalTo(1));
```

Comment:
Using `result()` not `actionResult()` in these tests, we don't care about the difference that makes to the exception that might be thrown.